build/patch/pre-autogen/libevent/common/000.patch

```diff
@@ -111,10 +82,10 @@ index 284341c..5b8b0a6 100644
 + */
 +#define _EVENT_HAVE_ARC4RANDOM_ADDRANDOM 1
```

Comment:
So, for sure the correct name of this define would be `EVENT__HAVE_ARC4RANDOM_ADDRANDOM`.